pandas/tests/series/test_timezones.py

```diff
@@ -364,3 +364,11 @@ def test_tz_localize_convert_copy_inplace_mutate(self, copy, method, tz):
                           index=date_range('20131027', periods=5, freq='1H',
                                            tz=tz))
         tm.assert_series_equal(result, expected)
+
+    def test_series_set_tz_timestamp(self):
+        # GH 25506
+        ts = Timestamp('2017-08-05 00:00:00+0100', tz=pytz.FixedOffset(60))
```

Comment:
can you use the tz_naive_fixture (tests a bunch of different timezones including fixed & no tz)